.github/workflows/lint-format-check.yml

```
@@ -0,0 +1,45 @@
+name: Formatting Checks and Markdown Linting
+
+on:
+  pull_request:
+    branches:
+      - main
+      - trunk
+
+jobs:
+  prettier-fmt-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "20"
```

Bump this back down to 18 - prettier isn't working with 20.

```suggestion
          node-version: "18"
```